keras/legacy_tf_layers/migration_utils.py

```diff
@@ -46,8 +46,11 @@ class DeterministicRandomTestTool(object):
     """
 
     def __init__(self, seed: int = 42, mode="constant"):
-        """Set mode to 'constant' or 'num_random_ops'. Defaults to
-        'constant'."""
+        """
+        Args:
+          mode: Set mode to 'constant' or 'num_random_ops'. Defaults to
+        'constant'.
+        """
```

Comment:
This docstring should be removed and added to the class docstrings.
Please also adde the seed docstring.

Reply:
Done; though I don't like the idea of the function documentation being on the `class`